doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -49,6 +49,10 @@ Board support
 * Added support for the Thingy:91 X board.
 * Updated Thingy:91 X board to use the ``nordic,pm-ext-flash`` node instead of external flash device name in static partitions.
 * Removed invalid external flash from static partitions for Thingy:91 X.
+* Following Zephyr RTOS ``native_posix`` has been deprecated in favour of ``native_sim``.
+
+  * Moved from ``native_posix`` to ``native_sim`` support in various tests and samples.
+  * ``native_posix`` will be removed with Zephyr 4.2 release.
```

Comment:
```suggestion
* Updated various tests and samples to use Zephyr's :ref:`native_sim<zephyr:native_sim>` instead of Zephyr's :ref:`native_posix<zephyr:native_posix>` for :ref:`running_unit_tests`.
  This mirrors the deprecation of ``native_posix`` in Zephyr.
  Support for ``native_posix`` will be removed in Zephyr with the v4.2 release.
```
If Zephyr v4.2 needs to be upmerged into sdk-nrf for this change to take effect, then the last sentences should be:
```
  Support for ``native_posix`` will be removed with the Zephyr v4.2 release.
  In the |NCS|, it will be removed once Zephyr v4.2 is upmerged to sdk-nrf.
```

Reply:
I confirm (from Vestavind) that the plan is that once native_posix is removed from upstream Zephyr in 4.2, NCS's sdk-zephyr will get that change in the next upmerge.